gax-httpjson/src/main/java/com/google/api/gax/httpjson/ManagedHttpJsonChannel.java

```diff
@@ -72,24 +73,31 @@ private ManagedHttpJsonChannel(
     this.httpTransport = httpTransport == null ? new NetHttpTransport() : httpTransport;
   }
 
+  @VisibleForTesting
```

Comment:
Use `@InternalApi("exposed for testing")`

Reply:
Done.